Learning_FEM_1D_step0.ipynb

```diff
@@ -580,7 +585,7 @@
     "\n",
     "We now only need to focus on the integration over a single element $K$:\n",
     "\n",
-    "$$\\int_{\\text{element}_K} \\, \\left( \\frac{db_{\\hat i}}{dx}D\\frac{db_{\\hat j}}{dx} + b_{\\hat i}\\sigma b_{\\hat j} \\right) = \\int_{\\text{element}_K} dx \\, \\left( b_{\\hat i}(x)\\, q \\right)$$\n",
+    "$$ \\underbrace{\\int_{\\text{element}_K} \\, \\left( \\frac{db_{\\hat i}}{dx}D\\frac{db_{\\hat j}}{dx} + b_{\\hat i}\\sigma b_{\\hat j} \\right)}_{\\textbf{Integration on the left hand side for element}_{K}} \\textbf{and} \\underbrace{\\int_{\\text{element}_K} dx \\, \\left( b_{\\hat i}(x)\\, q \\right)}_{\\textbf{Integration on the right hand side for element}_{K}}$$\n",
```

Comment:
same, no bold face

Reply:
Removed boldface, as per suggestion. Thank you very much.